specs/App.spec.js

```diff
@@ -1,11 +1,21 @@
-import React from 'react';
-import { render } from '@testing-library/react-native';
-
-import App from '../App';
+import React from 'react'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import { render } from '@testing-library/react-native'
+import App from '../App'
+import initialState from '../src/state/store/initialState'
 
 describe('<App />', () => {
-  it('has 1 child', () => {
-    const screen = render(<App />)
-    expect(screen.getByText('TuneShare').children).toHaveLength(1)
-  });
-});
\ No newline at end of file
+  const mockStore = configureStore([])
+  const store = mockStore(initialState)
+
+  it('displays app title in header', () => {
+    const screen = render(
+      <Provider store={store}>
+        <App />
+      </Provider>
+    )
+    expect(screen.getByText('TuneShare')).toBeTruthy()
+    expect(screen.getByText('TuneShare').props.style.fontSize).toEqual(17)
```

Comment:
This was an example in my demo commit. I see no reason to actually use this assertion.